controllers/v1/userExtension.js

```diff
@@ -5,15 +5,15 @@
  * Description : User creation and related information.
  */
 
-const userExtensionHelper = require(MODULES_BASE_PATH + "/user-extension/helper.js");
+const userExtensionHelper = require(MODULES_BASE_PATH + "/userExtension/helper.js");
 const csvFileStream = require(ROOT_PATH + "/generics/file-stream");
 
 /**
-    * userExtension
+    * UserExtension
```

Comment:
@rakeshSgr - Let's rename this to **"users"**

Reply:
resolved